vm/src/stdlib/re.rs

```diff
@@ -111,9 +111,8 @@ struct PyMatch {
 }
 
 impl PyValue for PyMatch {
-    fn required_type(_ctx: &PyContext) -> PyObjectRef {
-        // TODO
-        unimplemented!()
+    fn class(vm: &mut VirtualMachine) -> PyObjectRef {
+        vm.import("re").unwrap().get_attr("Match").unwrap()
```

Comment:
Maybe add a VirtualMachine `import_expect()` method that panics with a message like "expected module {:?} to be available"? It would cut out a lot of similar `.expect()` messages, and it's good to have a message because in theory it's not absolutely guaranteed that a given module would be available you're calling a type check or something.

Reply:
Sure, using the import here still feels weird to me though, hopefully we can figure out something better in the future.

Reply:
This is defenitely very weird. I also hope we can figure out a better way.